### PR TITLE
CORE and CCSM Specifications - Editorial Review - Ready for approval

### PR DESCRIPTION
**Pull request by Consianimis**
changes done:
--------------
- updated table of content
- updated introduction
- BPI User changed to BPI Subject
- BPI user account changed to BPI subject account
- updated acknowledgement section
- updated revision history section
- links added

in progress:
-------------
- started to update references, we need to follow ieee and w3c citations format appropriately
- for some reasons, link for appendix c in table of content does not work

### CODE REVIEW

File: DLT/baseline-dlt-v1.0-psd01.md [hunk 1]
```diff
@@ -372,130 +418,29 @@ Remove this note before submitting for publication.)
...
 
-| Revision | Date | Editor | Changes Made |
-| :--- | :--- | :--- | :--- |
-| baseline-core-v1.0-psd01 | 2020-09-29 | Anais Ofranc | Initial working draft |
+**Participants**:
 
+Andreas Freund \
+Anais Ofranc, Consianimis \
+Gage Mondok, Chainlink \
+Kyle Thomas, Provide Technologies Inc \
```
Comment (kthomas) [inline DLT/baseline-dlt-v1.0-psd01.md:433]: Just "Provide" here, please :)

File: api/baseline-api-v1.0-psd01.md [hunk 2]
```diff
@@ -26,17 +26,16 @@ https://docs.oasis-open.org/baseline/baseline-api/v1.0/baseline-api-v1.0.pdf
...
 #### Open Project:
-[Baseline, part of the Ethereum OASIS Open Project](https://www.baseline-protocol.org/)
+[Baseline, part of the EEA Community Projects](https://www.baseline-protocol.org/)
 
 #### Project Chair:
 John Wolpert (john.wolpert@mesh.xyz), [ConsenSys](https://consensys.net/) 
 
 #### Editors:
-Anais Ofranc (aofranc@consianimis.com), [Consianimis](https://www.consianimis.com/) \
+Kyle Thomas (kyle@provide.services), [Provide Technologies](https://provide.services/), \
```
Comment (kthomas) [inline api/baseline-api-v1.0-psd01.md:35]: Just "Provide" here, please :)

**Comment by Therecanbeonlyone1969** [comment]
@Consianimis ... can this be completed in the next 24 hours such that the TSC members can review? cc @kthomas @humbitious

**Comment by Consianimis** [comment]
@kthomas , @Therecanbeonlyone1969 - This PR is now for approval and merge.

**Review by Therecanbeonlyone1969 — APPROVED** [review]

**Review by kthomas — APPROVED** [review]

**Pull request merged**